test/unit/specs/dashboard/cd-dashboard-news.spec.js

```diff
@@ -0,0 +1,172 @@
+        { type: 'News', date: moment('2018-08-11T10:00:00.000Z'), link: 'blah3', title: 'blah3' },
+        { type: 'News', date: moment('2018-08-12T10:00:00.000Z'), link: 'blah4', title: 'blah4' },
+        { type: 'News', date: moment('2018-08-13T10:00:00.000Z'), link: 'blah5', title: 'blah5' },
+        { type: 'News', date: moment('2018-08-14T10:00:00.000Z'), link: 'blah6', title: 'blah6' },
+        { type: 'News', date: moment('2018-08-15T10:00:00.000Z'), link: 'blah7', title: 'blah7' }];
+
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.news = [
+        { date: '2018-08-09T10:00:00.000Z', link: 'blah1', title: { rendered: 'blah1' } },
+        { date: '2018-08-10T10:00:00.000Z', link: 'blah2', title: { rendered: 'blah2' } },
+        { date: '2018-08-11T10:00:00.000Z', link: 'blah3', title: { rendered: 'blah3' } },
+        { date: '2018-08-12T10:00:00.000Z', link: 'blah4', title: { rendered: 'blah4' } },
+        { date: '2018-08-13T10:00:00.000Z', link: 'blah5', title: { rendered: 'blah5' } },
+        { date: '2018-08-14T10:00:00.000Z', link: 'blah6', title: { rendered: 'blah6' } },
+        { date: '2018-08-15T10:00:00.000Z', link: 'blah7', title: { rendered: 'blah7' } }];
+
+        // ASSERT
+        expect(vm.formattedNews).to.deep.equal(mockFormattedNews);
+      });
+    });
+
+    describe('computed.isDisplayable', () => {
+      it('should return true if loadedPosts is true', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.loadedPosts = true;
+
+        // ASSERT
+        expect(vm.isDisplayable).to.equal(true);
+      });
+      it('should return false if loadedPosts is false', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.loadedPosts = false;
+
+        // ASSERT
+        expect(vm.isDisplayable).to.equal(false);
+      });
+    });
+  });
+
+  describe('methods', () => {
+    describe('methods.loadNews', () => {
+      it('should load the latest news', async () => {
+        // ARRANGE
+        const mockNews = [
+        { date: '2018-08-09T', link: 'blah1', title: { rendered: 'blah1' } },
+        { date: '2018-08-10T', link: 'blah2', title: { rendered: 'blah2' } },
+        { date: '2018-08-11T', link: 'blah3', title: { rendered: 'blah3' } }];
+        MockUpdatesService.loadNews.returns(Promise.resolve({ body: mockNews }));
+        const vm = vueUnitHelper(DashboardNewsComponentWithMocks);
+        vm.loadedPosts = false;
+
+        // ACT
+        await vm.loadNews(3);
+
+        // ASSERT
+        expect(vm.news).to.equal(mockNews);
+        expect(vm.loadedPosts).to.equal(true);
+      });
+    });
+
+    describe('methods.sortPostsByDate', () => {
+      it('should sort the list of posts by date', async () => {
+        // ARRANGE
+        const posts = [
+        { type: 'News', date: moment('2018-08-12T10:00:00.000Z'), link: 'blah1', title: 'blah1' },
```

Comment:
Your posts are already sorted. Inverse the array or use 3 of them in disorder to confirm it works the expected way

Reply:
woop yeah sorry i had taken the 3rd one out. will put it back ><